extensions/pass/src/passwordDetails.tsx

```diff
@@ -0,0 +1,168 @@
+    title: "Decrypting file",
+  });
+
+  cmd.stdout!.on("data", async (data) => {
+    toast.style = Toast.Style.Success;
+    toast.title = data;
+    await showHUD(data);
+  });
+
+  cmd.on("close", (code) => {
+    if (code != 0) {
+      toast.style = Toast.Style.Failure;
+      toast.title = "Failed to copy OTP";
+    }
+  });
+};
+
+function PasswordMetadata(props: passwords_path_structure) {
+  const ParseMetadata = (raw_data: string) => {
+    return raw_data.split("\n").map((val) => {
+      const indx = val.indexOf(":");
+      const field_name = val.slice(0, indx);
+      const field_value = val.slice(indx).substring(1);
+      return { name: field_name, value: field_value };
+    });
+  };
+
+  const loadPasswordDetails = () => {
+    const [markdown, setMarkdown] = useState<password_meta>([]);
+    options = {
+      ...options,
+      onData: (data: string) => {
+        setMarkdown(ParseMetadata(data));
+      },
+    };
+
+    const { isLoading } = useExec("pass", ["tail", `'${props.pass_file_name}'`], options);
+    return (
+      <List isLoading={isLoading}>
+        {markdown.map((val, i) => {
+          return (
+            <List.Item
+              key={i}
+              title={val.name}
+              accessories={[{ text: val.value }]}
+              actions={
+                <ActionPanel title={val.name}>
+                  <Action.CopyToClipboard title={`Copy value of '${val.name}'`} content={val.value} />
+                  <Action
+                    title={"Copy Password"}
+                    onAction={() => CopyPassword(props)}
+                    shortcut={{ modifiers: ["ctrl"], key: "c" }}
+                  />
+                  <Action
+                    title={"Copy OTP"}
+                    onAction={() => CopyOTP(props)}
+                    shortcut={{ modifiers: ["ctrl"], key: "o" }}
+                  />
+                </ActionPanel>
+              }
+            />
+          );
+        })}
+      </List>
+    );
+  };
+  return loadPasswordDetails();
+}
+
+export default function GetPasswordDetails(props: passwords_path_structure) {
+  return (
+    <ActionPanel>
+      <Action.Push
+        title={"Browse metadata"}
+        target={<PasswordMetadata pass_file_path={props.pass_file_path} pass_file_name={props.pass_file_name} />}
+        shortcut={{ modifiers: ["cmd"], key: "o" }}
+      />
+      <Action
+        title={"Copy Password"}
+        onAction={() => CopyPassword(props)}
+        shortcut={{ modifiers: ["ctrl"], key: "c" }}
+      />
+      <Action title={"Copy OTP"} onAction={() => CopyOTP(props)} shortcut={{ modifiers: ["ctrl"], key: "o" }} />
+      <Action.Push
+        shortcut={{ modifiers: ["ctrl"], key: "a" }}
+        title={"Create Pass Entry"}
+        target={<CreatePassForm />}
+      />
+      <Action
+        shortcut={{ modifiers: ["ctrl"], key: "d" }}
+        onAction={() => DeletePassword(props)}
+        title={"Delete Pass Entry"}
+      />
```

Comment:
We have special styling for [destructive actions](https://developers.raycast.com/api-reference/user-interface/actions#action.style), it may be worth using it here.